lint-staged.config.js

```diff
@@ -0,0 +1,19 @@
+//eslint-disable-next-line @typescript-eslint/no-var-requires
+const { ESLint } = require('eslint');
+
+const removeIgnoredFiles = async (files) => {
+	const eslint = new ESLint();
+	const ignoredFiles = await Promise.all(
+		files.map((file) => eslint.isPathIgnored(file))
+	);
+	const filteredFiles = files.filter((_, i) => !ignoredFiles[i]);
+	return filteredFiles.join(' ');
+};
+
+module.exports = {
+	'*.{js,ts}': async (files) => {
```

Comment:
Previously `jest --findRelatedTests` was also run, was it intended to remove that?

Reply:
I forgot to mention this! Yes, I did that on purpose indeed. That way, the pre-commit tasks are as fast as possible.